src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala

```diff
@@ -208,7 +208,7 @@ class RemoteServicesMockServer(port:Int) {
     mockServer.when(
       request()
         .withMethod("DELETE")
-        .withPath("/workflow/already_terminal_workflow")
+        .withPath("/workflow/v1/already_terminal_workflow")
```

Comment:
It can't be seen in these diffs, but the expectation here is incorrect.  Cromwell currently returns a 403 for an abort on an already-terminal workflow.